ts/components/AssistanceFormWebView.tsx

```diff
@@ -0,0 +1,75 @@
+import { StyleSheet, View } from "react-native";
+import React, { createRef, FC, useState } from "react";
+import WebView from "react-native-webview";
+import { AVOID_ZOOM_JS, closeInjectedScript } from "../utils/webview";
+import { RefreshIndicator } from "./ui/RefreshIndicator";
+import GenericErrorComponent from "./screens/GenericErrorComponent";
+import { withLightModalContext } from "./helpers/withLightModalContext";
+
+const URI = "http://localhost:5000/";
```

Comment:
Maybe this can help remember that is not ready:
```suggestion
// TODO: set the right uri
const URI = "http://localhost:5000/";
```
What do you think?